pkgs/by-name/ba/babl/package.nix

```
@@ -31,6 +31,17 @@ stdenv.mkDerivation (finalAttrs: {
     ./dev-prefix.patch
   ];
 
+  postPatch = ''
+    substituteInPlace meson.build \
+      --replace-fail "dependency('vapigen'," "dependency('vapigen', native: true,"
```

```suggestion
    substituteInPlace meson.build \
      --replace-fail "dependency('vapigen', version:'>=0.20.0'" "find_program('vapigen', native: true"
```
Would allow you to avoid putting a pkg-config in depsBuildBuild. Unless there is a reason to use `dependency` here.